docs/modules/ROOT/pages/tests.adoc

```diff
@@ -1,6 +1,6 @@
 = RAGStack Tests
 
-The latest RAGStack test reports are available https://ragstack-ai.testspace.com/projects/67980/spaces[here]{external-link-icon}.
+The latest RAGStack test reports are available https://ragstack-ai.testspace.com/[here]{external-link-icon}.
```

Comment:
Per the Google Developer Doc Style Guide, avoid using generic link text like "here".